pallets/xyk/src/lib.rs

```diff
@@ -675,6 +688,26 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		#[pallet::weight(<<T as Config>::WeightInfo>::multiswap_sell_asset(swap_token_list.len() as u32))]
```

Comment:
Missing docs:

Please add the docs to all `pub fn` for our docs autogeneration, the same as it is done in the Bootstrap pallet.